Libraries/graphics/Sources/Format/PNG.cpp

```diff
@@ -480,7 +480,7 @@ Write(Image const &image)
     strm.next_in = buffer.data();
 
     /* Maximum compressed size is x1.01 + 12 the uncompressed size. */
-    std::vector<uint8_t> compressed = std::vector<uint8_t>(buffer.size() * 1.01 + 12);
+    std::vector<uint8_t> compressed = std::vector<uint8_t>(static_cast<size_t>(buffer.size() * 1.01) + 12);
```

Comment:
All the code I've seen uses casts for it, so I assume down? I'm not quite sure I understand how this works but it's what the zlib tutorial did. ;P